docs/book/src/migration/multi-group.md

```diff
@@ -38,58 +41,72 @@ can check `api/v1/groupversion_info.go` to find that out:
 package v1
 ```
 
-Then, we'll rename `api` to `apis` to be more clear, and we'll move our
-existing APIs into a new subdirectory, "batch":
+Then, we'll rename move our existing APIs into a new subdirectory, "batch":
 
 ```bash
-mkdir apis/batch
-mv api/* apis/batch
-# After ensuring that all was moved successfully remove the old directory `api/`
-rm -rf api/ 
+mv api/* api/batch
```

Comment:
All steps still required rifght?
We need to create the dir with the group name
mv all api files to each and then remove api/

Reply:
We only need the mkdir, which I added back. The `mv api/* api/batch` will throw a warning that looks like `mv: cannot move 'batch' to a subdirectory of itself, 'batch/batch'`
 but that can be ignored safely.